docker/test/stateless/setup_minio.sh

```diff
@@ -101,7 +101,7 @@ upload_data() {
   # shellcheck disable=SC2045
   for file in $(ls "${data_path}"); do
     echo "${file}";
-    ./mc cp "${data_path}"/"${file}" clickminio/test/"${file}";
+    ./mc cp --recursive "${data_path}"/ clickminio/test/;
   done
```

Comment:
Actually, as we do recursive copy now, we don't need to iterate over files. We can change the whole loop to `./mc cp "${data_path}"/"${file}" clickminio/test/"${file}";`